M2/BUILD/mahrud/Makefile

```diff
@@ -0,0 +1,24 @@
+# 
+# Written for Fedora 26
+#
+
+.PHONY: always
+
+rpm-docker-build:
+	[ ! -z `docker images -q m2-rpm` ] || docker build -t m2-rpm .
+	docker run -it \
+	  -v `pwd`/../../..:/M2 \
+	  m2-rpm \
+	  make rpm-build -j4 -C /M2/M2/BUILD/mahrud
+
+rpm-build:
+	mkdir -p build;
+	make -j4 -C ../..;
+	cd build && ../../../configure \
+	  --enable-rpm \
+	  --prefix=/usr \
+	  --enable-download \
+	  --enable-build-libraries="factory pari flint ntl fflas_ffpack memtailor mathic mathicgb";
```

Comment:
This is the result of `make status`, in case it helps:
`# '/home/mahrud/Documents/Projects/M2/M2/M2/configure'  '--enable-encap' '--enable-download' '--enable-tarfile' '--enable-strip' '--enable-rpm' '--enable-build-libraries=' 'CC=' 'CXX=' 'FC=' 'CPPFLAGS=' 'LDFLAGS=' '--prefix=/usr' '--build=x86_64-pc-linux-gnu' 'build_alias=x86_64-pc-linux-gnu'`